tests/codex/testnode.nim

```diff
@@ -114,7 +103,7 @@ asyncchecksuite "Test Node":
 
     check:
       fetched.cid == manifest.cid
```

Comment:
Probably comparing the manifests directly is better?

```suggestion
      fetched == manifest
```

Reply:
You're right, fixing it.